src/main.rs

```diff
@@ -27,18 +27,23 @@ struct Args {
     #[arg(short, long)]
     work_dir: PathBuf,
 
-    /// The directory that will be copied to. Used to initialize source dir
+    /// The directory that will be copied to
     #[arg(short, long)]
     backup_dir: PathBuf,
+
+    // The directory that holds the original source files
+    #[arg(short, long)]
+    source_dir: PathBuf,
```

Comment:
what are your thoughts on making this optional, and using the first backup_dir if it isn't provided?

Reply:
agreed. I tried that initially. If that's possible then yes I agree. I can't do `source_dir = backup_dir.clone()` or whatever I tried so I wasn't sure how to get around that one. I'll give it a shot and if it doesn't work then I'd like for you to show me how to do it.